entrypoint/versionbump/componentsbump_test.go

```diff
@@ -17,13 +17,12 @@ const (
 	kgn               = "kubectl get node -o yaml"
 	canalFlannel      = kgn + " : | grep 'hardened-flannel' -A1, "
 	calico            = kgn + " : | grep 'hardened-calico' -A1, "
-	ingressController = kgn + " : | grep 'nginx-ingress-controller' -A1, "
+	ingressController = kgn + " : | grep 'nginx-ingress-controller' -A1"
```

Comment:
Dont we need a `, ` here after A1?

Reply:
cause i replaced the commands sequence to at least the first ones being the same on both products to give less trouble.....
so right now for both products the first 7 components are the same
coredns + metricsServer + etcd + cniPlugins + containerd + runc + crictl
 

so `ingressController` its the last one now, not needing the ` , `

Reply:
Can we add ingressController in the end then? That would make it easier to read th code.

Reply:
done